index.html

```diff
@@ -340,6 +324,10 @@ <h5 class="text-justify">CEE Alumnus Dr. Shao-Yuan (Ben) Leu (PhD ’09) promote
 							</p>
 						</div>
 					</div>
+<<<<<<< HEAD
```

Comment:
Check if this is needed here. This is causing issues in the footer and in the news recognition section.